lib/webauthn/authenticator_response.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "webauthn/authenticator_data"
```

Comment:
Is this for having access to the error constant?

Reply:
Yep